CHANGELOG.md

```diff
@@ -160,6 +160,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - IntegrationCLITestCase creates a new directory and chdir into it for each test
 - Automated testing of Automating Classification tutorial
 - `dffml version` command now prints git repo hash and if the repo is dirty
+- `dffml.high_level.ml.accuracy` to `dffml.high_level.ml.score`
```

Comment:
Let's put this in `unreleased`, 
```suggestion
- Renamed `accuracy()` to `score()`
```

Reply:
😅 my bad, added it here by mistake, would move it to unreleased and would also reword it as per your suggestion above